test/maps/market_trading.wmf/scripting/test_custom_units.lua

```diff
@@ -0,0 +1,19 @@
+run(function()
+   sleep(2000)
+   prefilled_buildings(p1, { "barbarians_custom_building", 22, 27 })
+   custom_building = map:get_field(22, 27).immovable
+   -- NOCOM busy roads are broken
```

Comment:
What do you mean by this?

Reply:
I can neither remember nor reproduce. Probably some trouble with an intermediate state of the code that is now gone. Will remove this comment.

Reply:
Then do please :)
This is the last nit, once this is done this is good to go in